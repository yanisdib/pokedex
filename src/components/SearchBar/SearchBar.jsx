import styled from 'styled-components';


const SearchBar = styled.input`
    width: 90%;
    min-height: 40px;
    height: 45px;
    background-color: #ffffff;
    border-radius: 50px;
`;


export default SearchBar;