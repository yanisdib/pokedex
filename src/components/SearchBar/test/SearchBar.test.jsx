import { shallow } from 'enzyme';
import SearchBar from '../SearchBar';

describe('SearchBar component', () => {
    let wrapper;

    beforeEach(() => wrapper = shallow(<SearchBar />))

    it('renders correctly without crashing', () => expect(wrapper).toMatchSnapshot())
})