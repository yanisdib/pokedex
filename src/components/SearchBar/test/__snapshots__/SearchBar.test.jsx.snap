// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBar component renders correctly without crashing 1`] = `ShallowWrapper {}`;
